admin/dashboard/assets/src/dashboard-app/pages/fonts/edit/EditLocalFont.js

```diff
@@ -372,6 +325,8 @@ const EditLocalFont = ({fontId}) => {
 				<p className="mb-5 text-xl font-semibold">
 					{__( 'Edit Font', 'custom-fonts' )}
 				</p>
+				<input className="w-full" type="text" onChange={(e) => handleInputChange(e,"font_name")} value={editFontData.font_name}></input>
+				<p>&nbsp;</p>
```

Comment:
Is this line added to add spacing ? @imnavanath

Reply:
Yes @nathanrodrigues2111